docs/workflow/building/coreclr/nativeaot.md

```diff
@@ -2,6 +2,21 @@
 
 The Native AOT toolchain can be currently built for Linux (x64/arm64), macOS (x64) and Windows (x64/arm64).
 
+## Building
+
+1. [Install pre-requisites](../../README.md#build-requirements)
+1. Run `build[.cmd|.sh] clr.nativeaot+libs -rc [Debug|Release]` from the repo root to build binaries for local development. This will build individual components, but not the NuGet packages and builds much faster.
+
+The paths to major components can be overriden using `IlcToolsPath`, `IlcSdkPath`, `IlcFrameworkPath`, `IlcFrameworkNativePath` and `IlcMibcPath` properties for `dotnet publish`. For example, `/p:IlcToolsPath=<repo root>\artifacts\bin\coreclr\windows.x64.Debug\ilc` can be used to override the compiler with a local debug build for troubleshooting or quick iterations.
```

Comment:
Would this paragraph make more sense in the "Building packages" section? It isn't immediately obvious these are instructions to consume the built bits.

Reply:
I added a new subsection header. I think it's better here, as this is specifically for referencing binaries, not packages.